### ISSUE TITLE
error in kmax setup command: use_2to3 is invalid.

### ISSUE THREAD

**Issue by garandria** [paulgazz/kmax#129]
I get the error in the title when running the following command:

```
$ sudo python3 setup.py develop
error in kmax setup command: use_2to3 is invalid.
```

It seems that [use_2to3](https://github.com/paulgazz/kmax/blob/4cda898643b8923b60447a9d54512f064af760dd/setup.py#L49) is [not supported anymore since setuptools v58.0.0](https://setuptools.pypa.io/en/latest/history.html#id166).

**Comment by paulgazz** [comment]
Thanks for your report.  Looks like I just need to do a one-time port with 2o3 so that use_2to3 will be unnecessary.  Will update and test this week.

**Comment by paulgazz** [comment]
In the meantime, you should be able to workaround this for now by using an older version of setuptools with `pip3 install setuptools==57.5.0`.  If you use a virtual environment, e.g., `python3 -m venv kmax_env`, you won't need to pollute your system's setuptools version.

**Comment by paulgazz** [comment]
Fixed by #154
